iib/workers/tasks/opm_operations.py

```diff
@@ -406,6 +406,11 @@ def opm_generate_dockerfile(
             log.info('Rewriting Dockerfile %s with newly generated by opm.', dockerfile_path)
         os.rename(dockerfile_path_opm_default, dockerfile_path)
 
+    local_cache_path = os.path.join(base_dir, 'cache')
+    if not os.path.exists(local_cache_path):
```

Comment:
Is there a case where the `local_cache_path` would already exist? If it does exist, how do we know if it is synced from the FBC we're about to build the image with?

Reply:
I'm pretty sure it's safe to just always run `/bin/opm serve /configs --cache-dir=/tmp/cache --cache-only`, even if the cache dir already exists. It will check cache integrity and only rebuild if hashes don't match.